packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -134,119 +198,138 @@ export const PlanDetails = ({
                       </Localized>
 
                       <Localized
-                        id={`coupon-amount`}
+                        id={`plan-price-${interval}`}
+                        data-testid="plan-price-total"
                         attrs={{ title: true }}
                         vars={{
-                          amount: getLocalizedCurrency(
-                            coupon.discountAmount,
-                            currency
-                          ),
+                          amount: getLocalizedCurrency(totalAmount, currency),
                           intervalCount: interval_count,
                         }}
                       >
-                        <div>
-                          {`- ${getLocalizedCurrencyString(
-                            coupon.discountAmount,
-                            currency
-                          )}`}
+                        <div
+                          className="total-price"
+                          title={totalPrice}
+                          data-testid="total-price"
+                          id="total-price"
+                        >
+                          {totalPrice}
                         </div>
                       </Localized>
                     </div>
-                  </div>
-                )}
-
-                <div className="plan-details-total-inner">
-                  <Localized id="plan-details-total-label">
-                    <div className="total-label">Total</div>
-                  </Localized>
-
-                  <Localized
-                    id={`plan-price-${interval}`}
-                    data-testid="plan-price-total"
-                    attrs={{ title: true }}
-                    vars={{
-                      amount:
-                        coupon && coupon.discountAmount
-                          ? getLocalizedCurrency(
-                              amount ? amount - coupon.discountAmount : amount,
-                              currency
-                            )
-                          : getLocalizedCurrency(amount, currency),
-                      intervalCount: interval_count,
-                    }}
-                  >
-                    <div
-                      className="total-price"
-                      title={planPrice}
-                      data-testid="total-price"
-                      id="total-price"
-                    >
-                      {planPrice}
-                    </div>
-                  </Localized>
-                </div>
-
-                {infoBoxMessage &&
-                  (infoBoxMessage.couponDurationDate ? (
-                    <div
-                      className="green-icon-text coupon-info"
-                      data-testid="coupon-success-with-date"
-                    >
-                      <img src={infoLogo} alt="" />
+                  )}
+                  {infoBoxMessage &&
+                    (infoBoxMessage.couponDurationDate ? (
```

Comment:
Not sure if this is any easier w/ an [optional chaining (`?.`) operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) (but saves a whopping 1 `if` check): 
```js
{ infoBoxMessage?.couponDurationDate ? (…) : (…) }
```

Reply:
So this was the logic from before, this only got updated because of the linter ran when I made other edits to this file. Since it is not part of the scope of the work, I'd prefer just to leave the logic as is rather than have to potentially clean up the tests around this as well.